plenum/server/replica.py

```diff
@@ -204,7 +210,7 @@ def __init__(self, node: 'plenum.server.node.Node', instId: int,
         # viewNo and value a map of pre-prepare sequence number to commit
         self.stashed_out_of_order_commits = {}  # type: Dict[int,Dict[int,Commit]]
 
-        self.checkpoints = SortedDict(lambda k: k[0])
+        self.checkpoints = SortedDict(lambda k: k[1])
```

Comment:
Need only pp_seq_no, since gc is done after every `h` transactions so only `pp_seq_no` is relevant